src/obfl-specification.html

```diff
@@ -1557,18 +1562,121 @@ <h4 id="L735" class="include">table-of-contents</h4>
       </dl>
     </dd>
   <dt>Content Model</dt>
-    <dd>toc-entry [1 or more]</dd>
+    <dd>toc-block [1 or more]</dd>
+</dl>
+</div>
+
+<h4 id="L735" class="include">toc-block</h4>
+
+<p>The toc-block element defines a block of text that can be used within a
+table of contents. It is an extension of the block element with toc-entries.</p>
+
+<div class="sidebar markup">
+<dl>
+  <dt>Usage</dt>
+    <dd>table-of-contents, toc-block</dd>
+  <dt>General Attributes</dt>
+  <dd><p>This element supports the <a href="#block-atts">Block</a>, <a href="#margin-atts">Margin</a>
+    and <a href="#padding-atts">Padding</a> attributes groups.</p>
+      <p>This element supports the <a href="#borders">Borders</a> attribute
+      group.</p>
+    </dd>
+  <dt>Special Attributes</dt>
```

Comment:
Rather than copying the whole section I would create a new attribute group. But see my comment above, maybe we can skip the special attributes for now because `toc-entry` currently also doesn't have them.